renku/ui/cli/remove.py

```diff
@@ -17,16 +17,25 @@
 # limitations under the License.
 """Remove a file, a directory, or a symlink.
 
-Removing a file that belongs to a dataset will update its metadata. It also
-will attempt to update tracking information for files stored in an external
+Description
+~~~~~~~~~~~
+
+Remove a file that belongs to a dataset and update its metadata. It also
```

Comment:
This command is for removing any file (not just ones belonging to a dataset). but if they are in a dataset, it takes care of not having inconsistent metadata.